sympy/polys/tests/test_polytools.py

```diff
@@ -4082,3 +4082,88 @@ def test_issue_28156():
     )
     roots = rhs.ground_roots()
     assert roots
+
+def test_routh_hurwitz_stability():
```

Comment:
There should be two blank lines between top-level functions as in the rest of the file.